README.md

```diff
@@ -56,7 +56,9 @@ The following Visual Studio Code settings are available for the Go extension.  T
 	"go.formatFlags": [],
 	"go.goroot": "/usr/local/go",
 	"go.gopath": "/Users/lukeh/go",
-	"go.gocodeAutoBuild": false
+	"go.gocodeAutoBuild": false,
+	"go.runDir": "${file_dir}",
+	"go.runArgs": []
```

Comment:
I think this brings the feature a little close to overlapping with the task system. It seems highly unlikely that a single project-scoped argument array for a main will be applicable to every main you might try to run with an ad-hoc "run main at cursor" command. If you need to statically define arguments and stuff, there's a robust task runner system already in place.